docker/airrflow/Version.yaml

```diff
@@ -1,22 +1,18 @@
 immcantation:
-  version:  4.4.0
-  date:     2024.01.09
+  version:  devel
+  date:     2025.03.31
 package:
-  presto:   0.7.1
-  changeo:  1.3.0
   alakazam: 1.3.0
   shazam:   1.2.0
-  tigger:   1.0.1
-  scoper:   1.2.1
-  dowser:   1.2.0
-  enchantr: 0.1.16
-  igphyml:  1.1.5
+  scoper:   1.3.0
+  dowser:   2.3
+  enchantr: 0.1.21
+  amulety:  1.0
```

Comment:
Is amulety being installed in the container? We will need to add it to `docker/base/tools/versions.py`

Reply:
Ah sorry it's not being installed, I'll delete this row. I'm relying on the biocontainers container for the amulety part in airrflow